primitives/npos-elections/src/phragmms.rs

```diff
@@ -26,7 +26,7 @@ use crate::{
 	VoteWeight, Voter,
 };
 use sp_arithmetic::{traits::Bounded, PerThing, Rational128};
-use sp_std::{prelude::*, rc::Rc};
+use sp_std::{prelude::*, rc::Rc, vec};
```

Comment:
```suggestion
use sp_std::{prelude::*, rc::Rc};
```

Vec must already be in prelude 🤔